src/runtime_src/driver/xclng/drm/xocl/userpf/qdma.c

```diff
@@ -123,10 +123,10 @@ static int xocl_user_qdma_probe(struct pci_dev *pdev,
 
 	conf = &qd->dev_conf;
 	memset(conf, 0, sizeof(*conf));
-	conf->poll_mode = 0;
 	conf->pdev = pdev;
 	conf->intr_rngsz = QDMA_INTR_COAL_RING_SIZE;
-	conf->master_pf =  PCI_FUNC(pdev->devfn);
+	conf->master_pf = 1;
```

Comment:
Why master_pf is forced to 1? Using PCI_FUNC(pdev->devfn) should be better.

Reply:
It is a bit field, to indicate to libqdma if this dev is the qdma master pf (i.e., controls the qdma global csr space). Since there is only 1 qdma pf, so it is hard coded to 1.